drift-client/src/main/java/com/facebook/drift/client/DriftClientFactory.java

```diff
@@ -71,6 +71,7 @@ public DriftClientFactory(
         this.addressSelector = requireNonNull(addressSelector, "addressSelector is null");
         this.exceptionClassifier = exceptionClassifier;
         this.methodInvocationStatsFactory = requireNonNull(methodInvocationStatsFactory, "methodInvocationStatsFactory is null");
+        this.retryService = new BoundedExecutor(newCachedThreadPool(daemonThreadsNamed("drift-client-retry-service-%s")), 2 * Runtime.getRuntime().availableProcessors());
```

Comment:
nit : why 2x ?

Reply:
Currently the same amount of work is being handled by the eventLoopGroup which is 2x - and hence I didn't see a good reason to take it to 4x. I will probably go back and question the 4x to begin with :)